app/gui/view/graph-editor/src/component/visualization/foreign/java_script.rs

```diff
@@ -10,6 +10,10 @@ pub mod definition;
 pub mod instance;
 pub mod source;
 
-pub use definition::*;
-pub use instance::*;
+pub use definition::field;
+pub use definition::method;
+pub use definition::Definition;
+pub use definition::FallibleDefinition;
+pub use instance::Instance;
+pub use instance::PreprocessorCallback;
```

Comment:
I was forced to do that because of new lints `ambiguous_glob_reexports` and `silent_glob_reexports`. There are a lot of similar places that were affected, and my approach differs from case to case. We need to be far more cautious with glob reexports. Right now, it is tough to understand the source of the particular type.